docker-compose.yml

```diff
@@ -49,6 +48,8 @@ services:
       - products:/mnt/podata/products # .sto
       - product_images:/mnt/podata/product_images # .jpg
       - orgs:/mnt/podata/orgs # .sto
+      - html_data:/mnt/podata/html_data # Static data (e.g dumps)
```

Comment:
Do we need this to be a separate volume ? Is this a zfs clone also ?

Reply:
Also if we add it here, we must add it to deployement file.

Reply:
I don't think it's a ZFS clone, we need it to be writable by the backend.

Reply:
ok but all /mnt/podata should be writable. So we don't need a specific mount point here.

Reply:
ok, I removed it, it works.